client/src/pages/WelcomePage.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import { ReactComponent as Logo } from '../image/Logo.svg';
+
+export default function WelcomePage() {
+  return (
+    <PageWrapper>
+      <HeadLiner>Welcome to</HeadLiner>
+      <Logo />
+      <AppName>Crypto Starter</AppName>
+
+      <AppUsecases>Latest News</AppUsecases>
+      <AppUsecases>Save important Articles</AppUsecases>
+      <AppUsecases>Create Wallets</AppUsecases>
+      <StartTheApp>
+        Make your first experience
+        <Link to="/news"> here</Link>
+      </StartTheApp>
+    </PageWrapper>
+  );
+}
+
+const PageWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+
+  svg {
+    filter: drop-shadow(0rem 0rem 1rem var(--primary));
+    margin-top: 2.5rem;
+  }
+`;
+
+const HeadLiner = styled.h1``;
```

Comment:
if you do not apply any styles to a styled component, you do not need it (: ✨  you can just use <h1></h1> instead

Reply:
fixed :)